public/app/features/dashboard/components/DashNav/DashNav.tsx

```diff
@@ -306,7 +307,7 @@ export const DashNav = React.memo<Props>((props) => {
       );
     }
 
-    if (showSettings) {
+    if (showSettings && !dashboard.meta.isEmbedded) {
```

Comment:
we need the settings views, they are essential for so many dashboard settings, queries, annotations, links, variables

Reply:
They are not needed for the editor embedded dashboard.

Reply:
????

Reply:
Hmm, am I missing something here? `dashboard.meta.isEmbedded` will be only set if the URL is `d-embed`, otherwise it's `false`. For the editor embedded dashboard we don't need the settings options, in fact we need to disable all the actions that can lead to other Grafana pages than `d-embed`.

When it's a normal dashboard display view at `/d/[UID]/[NAME]`, `dashboard.meta.isEmbedded` will be false, so the `showSettings` toggle will work as before.

Reply:
Maybe this screenshot will help to clarify things. This is how embedded Grafana looks inside the VS Code editor currently (note, there's not even a URL bar). We only need the existing functionality in the screenshot to preview a dashboard from JSON, and that's what `dashboard.meta.isEmbedded` is meant in this case. I'll rename it to `dashboard.meta.isEditorEmbedded` to better clarify its purpose. 
![Screenshot 2023-08-22 at 14 38 53](https://github.com/grafana/grafana/assets/8878045/fc622202-80ec-474c-a85f-1c94cce5d1c8)

Reply:
I just fail completely to understand the thinking behind this, what is the point of this page if your can't edit dashboard settings, title, time range options, variables, annotations, dashboard links etc? 

It's not possible to remove all links to other places, data links will link to other dashboards (as will dashboard links). This all seem very strange to me